spec/minimax_server/router_spec.clj

```diff
@@ -0,0 +1,37 @@
+(ns minimax-server.router-spec
+  (:import [scarvill.httpserver.response Response]
+           [scarvill.httpserver.request Request RequestBuilder Method])
+  (:require [speclj.core :refer :all]
+            [minimax-server.router :refer :all]
+            [clojure.string :as str]))
+
+(describe "router"
+
+  (defn request-with [current-player board]
+    (-> (new RequestBuilder)
+        (.setParameter "current_player" current-player)
+        (.setParameter "board" board)
+        (.setURI "/")
+        .build))
+
+  (defn get-body-as-str [response]
+    (String. (.getBody response)))
+
+  (def example-request (request-with "x" "x,o,x,o,x,o,x,o,_"))
+
+  (it "returns a response"
+     (should-be-a Response (.apply (minimax-router (fn [_ _] "foo")) example-request)))
+
+  (it "returns the result of evaluating given function in the body of the response"
+      (should= "foo"
+               (get-body-as-str (.apply (minimax-router (fn [_ _] "foo")) example-request))))
+
+  (it "parses the current player from request parameters"
```

Comment:
We test that the correct move is returned in our computer_player.clj file. In router.clj, we are testing that the query data is being used/parsed appropriately.